### PR TITLE
Improve hover on button and h1 readablility

### PR DESCRIPTION
**Pull request by sajjadcodes**
I have just improved the readability of heading as well as a made href more interactive.

### CODE REVIEW

**Comment by sajjadcodes** [comment]
Improvement in readibility

File: stylesheets/stylesheet.css [hunk 1]
```diff
@@ -72,7 +72,7 @@ h1, h2, h3 {
 }
 
 h1 {
-  font-size: 56px;
+  font-size: 58px;
```
Comment (ritwik12) [inline stylesheets/stylesheet.css]: @sajadhussain Please remove this as it is not having any valid changes.

**Comment by ritwik12** [comment]
@sajadhussain Would like to keep the h1 as original only and please remove that change from this PR. I would like to keep hover in this PR as it looks good. @shiftkey Hover looks good but you can decide the exact px and size of hover.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: stylesheets/stylesheet.css (hunk 1 above)
Comment (sajjadcodes) [reply stylesheets/stylesheet.css]: okay got

**Comment by shiftkey** [comment]
> @shiftkey Hover looks good but you can decide the exact px and size of hover.

@ritwik12 @sajadhussain I'm curious what problem we're trying to address here with this change. I can see what's changed but I'd like to understand _why_ we're doing it this way, as we might be able to provide useful feedback about the changes or suggest a better solution.

**Comment by ritwik12** [comment]
@sajadhussain Please let us know what problems it will solve and why is it needed. Better to have an issue for this to discuss further.

**Comment by sajjadcodes** [comment]
It is UX improvement of the site. Just improve the button. code hover so it will more interactive and give a sense of working.

**Comment by ritwik12** [comment]
@sajadhussain Thanks for your PR, it might seem good but is not that much needed here as the primitive site looks good in itself.

We have many issues that need attention and would be happy if you can help there based on your skills. like #930, etc

@shiftkey Please feel free to give your thoughts here

**Comment by shiftkey** [comment]
> Please feel free to give your thoughts here

There's a few things I don't like about this PR, and I've tried to capture this in a GIF so you can see it for yourself:

 - when you hover over the button, the text alongside it gets nudged down slightly because of the padding changes. 



 - making the font size slightly bigger at the same time feels a bit jarring

 - this solution leverages `:hover`, which is only available for desktop users - what about mobile users, or users with screen readers?

> It is UX improvement of the site. Just improve the button. code hover so it will more interactive and give a sense of working.

In situations like this I'd prefer to discuss the problem through an issue, rather than going straight to a proposed solution using a pull request. The underlying problem is still unclear to me, and I believe using animations like this for web pages should only occur when absolutely necessary.

If we've got a good grasp on the problem and nothing else seems suitable, then I'm fine to revisit this.

**Pull request closed without merging**